src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -4,39 +4,54 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import seedu.address.commons.util.DateUtil;
 import seedu.address.model.AddressBook;
 import seedu.address.model.ReadOnlyAddressBook;
 import seedu.address.model.person.Address;
 import seedu.address.model.person.Email;
+import seedu.address.model.person.LoanRecords;
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
 import seedu.address.model.tag.Tag;
 
+
 /**
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
     public static Person[] getSamplePersons() {
+        LoanRecords tempLoanRecords = new LoanRecords();
+        try {
+            tempLoanRecords.addLoan(100, DateUtil.parse("2020-01-01"), DateUtil.parse("2020-01-02"));
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends")),
+                getTagSet("friends"),
+                tempLoanRecords),
             new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
                 new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
-                getTagSet("colleagues", "friends")),
+                getTagSet("colleagues", "friends"),
```

Comment:
Are tags complusory?

Reply:
This is from the default sample given by them, so I don't think there's a need to change it. Only relevant change to this is the addition of LoanRecords entry for each sample person.